src/Symfony/Component/Form/Extension/Core/DataTransformer/DateIntervalToStringTransformer.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+/**
+ * Transforms between a date string and a DateInterval object.
+ *
+ * @author Steffen Roßkamp <steffen.rosskamp@gimmickmedia.de>
+ */
+class DateIntervalToStringTransformer implements DataTransformerInterface
+{
+    /**
+     * Format used for generating strings.
+     *
+     * @var string
+     */
+    private $format;
+
+    /**
+     * Whether to parse by as a signed interval.
+     *
+     * @var bool
```

Comment:
phpdocs on private properties are useless here, you already document them in the constructor